compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ComposeWindow.uikit.kt

```diff
@@ -665,31 +665,35 @@ internal actual class ComposeWindow : UIViewController {
             }
         }
 
-        scene.setContent(
-            onPreviewKeyEvent = inputServices::onPreviewKeyEvent,
-            onKeyEvent = { false },
-            content = {
-                CompositionLocalProvider(
-                    LocalLayerContainer provides view,
-                    LocalUIViewController provides this,
-                    LocalKeyboardOverlapHeightState provides keyboardOverlapHeightState,
-                    LocalSafeArea provides safeAreaState,
-                    LocalLayoutMargins provides layoutMarginsState,
-                    LocalInterfaceOrientationState provides interfaceOrientationState,
-                    LocalSystemTheme provides systemTheme.value,
-                    LocalUIKitInteropContext provides interopContext,
-                ) {
-                    content()
-                }
-            },
-        )
-
+        fun onReadyToSetContent() {
+            scene.setContent(
+                onPreviewKeyEvent = inputServices::onPreviewKeyEvent,
+                onKeyEvent = { false },
+                content = {
+                    CompositionLocalProvider(
+                        LocalLayerContainer provides view,
+                        LocalUIViewController provides this,
+                        LocalKeyboardOverlapHeightState provides keyboardOverlapHeightState,
+                        LocalSafeArea provides safeAreaState,
+                        LocalLayoutMargins provides layoutMarginsState,
+                        LocalInterfaceOrientationState provides interfaceOrientationState,
+                        LocalSystemTheme provides systemTheme.value,
+                        LocalUIKitInteropContext provides interopContext,
+                    ) {
+                        content()
+                    }
```

Comment:
```suggestion
                        content = content
                    )
```

Reply:
Could you please comment your idea? Why asign it to itself? 
did you mean something else?

Reply:
I see, it's an argument. not an assignment